QModManager/Checks/PirateCheck.cs

```diff
@@ -29,20 +31,20 @@ internal static void PirateDetected()
 
         internal static void IsPirate(string folder)
         {
-            string steamDll = Path.Combine(folder, "steam_api64.dll");
+            var steamDll = Path.Combine(folder, "steam_api64.dll");
             if (File.Exists(steamDll))
             {
-                FileInfo fileInfo = new FileInfo(steamDll);
-
-                if (fileInfo.Length > 220000)
+                FileInfo dllInfo = new FileInfo(steamDll);
+                if (dllInfo.Length > FileSize)
                 {
-                    PirateDetected();
-                    return;
+                    FileChecking(folder);
                 }
```

Comment:
This way makes it so if a NEW crack is made that does not match our checks but still has the larger steam api then it will not flag at all. 

I would like to request that the file check be moved back outside of this check as this check SHOULD flag someone as a pirate if this dll is the wrong size as even if they reinstall over the cracked game this api file will have been fixed.